File: condicoes/1036.py
#1036 FÓRMULA DE BHASKARA
import math

valores = input().split(" ")

a = float(valores[0])
b = float(valores[1])
c = float(valores[2])

discriminante = math.pow(b,2) - (4*a*c)

if (discriminante >= 0 and a != 0):
    raiz1 = (-b + (math.sqrt(discriminante)))/(2*a)
    raiz2 = (-b - (math.sqrt(discriminante)))/(2*a)
    print("R1 = {:.5f}" .format(raiz1))
    print("R2 = {:.5f}" .format(raiz2))
else:
    print("Impossivel calcular")

File: javascript/1036.js
/*Fórmula de Bhaskara*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split(" ");

let A = parseFloat(lines.shift());
let B = parseFloat(lines.shift());
let C = parseFloat(lines.shift());

let Calculo = Math.pow(B, 2) - (4 * A * C);

if(Calculo >= 0 && A !== 0) {
  let Raiz = (-B + Math.sqrt(Calculo)) / (2 * A);
  let RaizQ = (-B - Math.sqrt(Calculo)) / (2 * A);

  console.log("R1 = " + Raiz.toFixed(5));
  console.log("R2 = " + RaizQ.toFixed(5));
}else{
  console.log("Impossivel calcular");
}